### ISSUE TITLE
Dynamic Text Box

### ISSUE THREAD

**Issue by brunocalado** [drewg13/foundryvtt-scum-and-villainy#37]
Would be possible to replace the text boxes for standard foundry journals?

This would make it auto cut space and also expand when required.

![image](https://user-images.githubusercontent.com/4999783/140435412-70adf8e0-ecea-4705-8943-0cd1de76aabe.png)

**Comment by drewg13** [comment]
I have no idea how to do that.  All of those fields are manually tweaked in the HTML templates because I dislike needing scrollbars.  I don't know of a way to make them dynamically size themselves.

**Comment by brunocalado** [comment]
The pbta system does that. If you want you can check it out.

https://user-images.githubusercontent.com/4999783/140614850-fddc1c35-c577-4e95-8a38-84f1f9bbea3b.mp4

Manifest: https://gitlab.com/asacolips-projects/foundry-mods/pbta/-/jobs/artifacts/0.3.0/raw/system.json?job=build-prod

**Comment by drewg13** [comment]
Yeah, that's using the TinyMCE editor instead of HTML textareas.  I don't love using TinyMCE more than necessary, as it adds complexity and has a history of losing data unexpectedly, so I don't trust it 100%.  I'll think about it, but I'm not sure I'm interested in changing all the textareas just for cosmetic improvement without a better solution than using more TinyMCE.

**Comment by brunocalado** [comment]
My user experience with loose data with tiny mce is when to users do
something at the same time in an actor sheet.

The player was writing his background and I updated another part of the
actor sheet. He lost his Unsaved work.

But, just that.

Em sáb, 6 de nov de 2021 14:29, drewg13 ***@***.***> escreveu:

**Comment by drewg13** [comment]
I'm coming around to this idea, slowly converting things over where they make the most sense.
